eng/Subsets.props

```
@@ -31,6 +31,7 @@
 
     <!-- Android 32-bit builds blocked by https://github.com/dotnet/runtime/issues/111665 -->
     <_CoreCLRSupportedOS Condition="'$(TargetsAndroid)' == 'true' and '$(TargetArchitecture)' != 'arm' and '$(TargetArchitecture)' != 'x86'">true</_CoreCLRSupportedOS>
+    <_CoreCLRSupportedOS Condition="'$(TargetOS)' == 'browser' and $(Subset.ToLowerInvariant().Contains('clr'))">true</_CoreCLRSupportedOS>
```

```suggestion
    <_CoreCLRSupportedOS Condition="'$(TargetsBrowser)' == 'true'</_CoreCLRSupportedOS>
```
I do not think this needs to be a fine grained based on what happens to work at the given moment.